modules/rosa-sdpolicy-storage.adoc

```diff
@@ -11,20 +11,20 @@ This section provides information about the service definition for {product-titl
 
 [id="rosa-sdpolicy-encrytpted-at-rest-storage_{context}"]
 == Encrypted-at-rest OS and node storage
-Control plane, infrastructure, and worker nodes use encrypted-at-rest AWS Elastic Block Store (EBS) storage.
+Control plane, infrastructure, and worker nodes use encrypted-at-rest Amazon Elastic Block Store (EBS) storage.
 
 [id="rosa-sdpolicy-encrytpted-at-rest-pv_{context}"]
 == Encrypted-at-rest PV
 EBS volumes that are used for PVs are encrypted-at-rest by default.
 
 [id="rosa-sdpolicy-block-storage_{context}"]
 == Block storage (RWO)
-Persistent volumes (PVs) are backed by AWS EBS, which is Read-Write-Once.
+Persistent volumes (PVs) are backed by Amazon EBS, which is Read-Write-Once.
 
 PVs can be attached only to a single node at a time and are specific to the availability zone in which they were provisioned. However, PVs can be attached to any node in the availability zone.
 
 Each cloud provider has its own limits for how many PVs can be attached to a single node. See link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#instance-type-volume-limits[AWS instance type limits] for details.
 
 == Shared Storage (RWX)
 
-The AWS CSI Driver can be used to provide RWX support for {product-title}. A community Operator is provided to simplify setup. See link:https://access.redhat.com/articles/5025181[AWS EFS Setup for OpenShift Dedicated and Red Hat OpenShift Service on AWS] for details. 
+The AWS CSI Driver can be used to provide RWX support for {product-title}. A community Operator is provided to simplify setup. See link:https://access.redhat.com/articles/5025181[Amazon EFS Setup for OpenShift Dedicated and Red Hat OpenShift Service on AWS] for details. 
```

Comment:
I have update the instance here. Please let me know if I need to revoke this instance. Thank you!

Reply:
I didn't heard any concern from @codymant @xueli181114 @duanwei33 to update the instances for link text. So, I am proceeding to keep this updates and resolving this comment. Similarly, I am resolving the comments that I have made for the link text clarification. Please let me know if you have other thoughts. Thank you!

Reply:
Revoked. Could you please confirm?

Reply:
Updated with, "Amazon Elastic File Storage" as this is the first instance in a module.